src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -20,10 +20,17 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
   private implicit val ec: ExecutionContext = actorRefFactory.dispatcher
 
   lazy val rawlsCuratorUrl = FireCloudConfig.Rawls.authUrl + "/user/role/curator"
+  val duosAutocompleteUrl = FireCloudConfig.Duos.baseUrl + "/autocomplete?types=disease&q=%s"
 
   val libraryServiceConstructor: UserInfo => LibraryService
 
   val libraryRoutes: Route =
+    path("duos-autocomplete" / Segment) { (searchTerm) =>
```

Comment:
nit: I'd make this `"duos" / "autocomplete"` ... that way, we can add more endpoints under `library/duos/` in a nice consistent way.